tests/integration/targets/mso_backup/tasks/main.yml

```diff
@@ -0,0 +1,295 @@
+# Test code for the MSO modules
+# Copyright: (c) 2020, Lionel Hercot (@lhercot) <lhercot@cisco.com>
```

Comment:
Remove me and Dag from this file. I don't think we still use much of the shared code.